cookbook/core/extend_flyte/custom_task_plugin.py

```diff
@@ -9,9 +9,13 @@
 extended to all users of Flyte, or there is a cost/visibility benefit of doing so. To understand when you should write
 a backend plugin refer to - TODO: Flow chart that helps you decide when to write a backend / flytekit plugin?
 
```

Comment:
Will we add a flowchart here?

Reply:
The flowchart is here: https://docs.google.com/presentation/d/1kKGR67oKVPunwOj-qCCPyDlmeBSKFftBUG-VpnJXCyM/edit#slide=id.gfb06fbb202_0_358

can you take this and save it as an SVG here?

Reply:
The flowchart was added to the "Extending Flyte" page using https://github.com/flyteorg/flytesnacks/pull/481.